### ISSUE TITLE
Add this to Terraform's official list of providers

### ISSUE THREAD

**Issue by jcs224** [squat/terraform-provider-vultr#5]
This looks like the most active attempt at creating a solid Vultr provider for Terraform. Would you be willing to submit a PR to hashicorp/terraform?

Here are the instructions for making the PR, including testing and documentation instructions:
https://github.com/hashicorp/terraform/blob/master/.github/CONTRIBUTING.md#new-provider

We are discussing this now:
https://github.com/hashicorp/terraform/issues/2611

**Comment by squat** [comment]
@jcs224 yes, I would be very happy to contribute this to the terraform-providers org. I have begun working on acceptance tests, which is the primary constraint right now since all resources require acceptance tests before a provider can be accepted. I'll continue working on these now and update this issue as they progress. Adding thorough tests will be an undertaking, so I would be happy to accept any contributions in the form of tests as that would speed up this process.

**Comment by dalenoe** [comment]
I would also really like to see this. I really wish this was an "official" provider. I know it is being worked on, just shocked it doesn't already exist. I wonder how long the digital ocean provider has been "official"

**Comment by danielsreichenbach** [comment]
@squat going to work on adding acceptance tests for this one, documentation needed for the terraform website is being worked on too (see [this branch](https://github.com/kogitoapp/terraform-provider-vultr/tree/documentation-updates))

**Comment by blaggacao** [comment]
I assume your heavily using this yourself, and it seems to be the most promising candidate for official support. I probably will early adapt, unless you have a warning for me, besides the usual "unofficial" tag. BR

**Comment by aequasi** [comment]
Any update on this? Anything the community can help with?

**Comment by squat** [comment]
@aequasi thank you for your enthusiasm! The things that still need love are:
1. docs + build; this is low-hanging and really just needs a rebase https://github.com/squat/terraform-provider-vultr/pull/27
2. acceptance tests; I have begun working on this but there is plenty of work here and I would love some help :)

**Comment by afady** [comment]
@squat can we keep a running list somewhere of which tests are complete or being worked on?

**Comment by jrab66** [comment]
i would love to work in docs :)

**Comment by OJFord** [comment]
@squat Looks like @afady's blitzed the acceptance tests; #27 still needs rebasing, but anything else we can help with?

I noticed it doesn't work with the upcoming terraform 0.12 release, but I'm not sure if that's a requirement for being added, or how hard it would be to fix (maybe it's just a case of claiming to support that version, and verifying tests pass).

**Comment by OJFord** [comment]
I guess it's more likely that this one gets added:
https://github.com/vultr/terraform-provider-vultr

Blog announcement:
https://www.vultr.com/news/Collaborate-with-Vultr-on-GitHub/
